ccmlib/scylla_node.py

```diff
@@ -1204,19 +1204,18 @@ def _write_agent_log4j_properties(self, agent_dir):
         raise NotImplementedError('ScyllaNode._write_agent_log4j_properties')
 
     def _wait_no_pending_flushes(self, wait_timeout=60):
-        def no_pending_flushes():
-            stdout, _ = self.nodetool('cfstats', timeout=wait_timeout)
+        def no_pending_flushes() -> bool:
+            stdout = self.nodetool('cfstats', timeout=wait_timeout)[0]
             pending_flushes = False
             for line in stdout.splitlines():
                 line = line.strip()
                 if line.startswith('Pending flushes'):
-                    _, pending_flushes_str = line.split(':')
+                    pending_flushes_str = line.split(':', maxsplit=1)[1]
                     pending_flushes_count = int(pending_flushes_str.strip())
                     if pending_flushes_count > 0:
                         pending_flushes = True
             return not pending_flushes
-        result = wait_for(no_pending_flushes, timeout=wait_timeout, step=1.0)
-        if result is None:
+        if wait_for(no_pending_flushes, timeout=wait_timeout, step=1.0) is None:
```

Comment:
why not `if not wait_for`?

Reply:
Right! If you fix wait_for using one of my two recommended fixes in the previous patches, it won't be None here. But even if you don't fix it, "if not wait_for" will also work because "None" is considered false. And "if not" is clearer.